server/app/routes/api/wines.js

```diff
@@ -0,0 +1,102 @@
+var express = require('express');
+var router = express.Router();
+var mongoose = require('mongoose')
+
+router.get('/', function(req, res) {
+	mongoose.model('Wine')
+	.find({})
+	.then(function(wines) {
+		res.json(wines)
+	})
+	.then(null, function(err) {
+		console.log(err)
+	})
+})
+
+router.get('/:id', function(req, res) {
+	var id = req.params.id
+	mongoose.model('Wine')
+	.findById(id)
+	.then(function(wine) {
+		res.json(wine)
+	})
+	.then(null, function(err) {
+		console.log(err)
+		res.sendStatus(404)
+	})
+})
+
+router.get('/:id/reviews', function(req, res) {
+	var id = req.params.id
+	mongoose.model('Wine')
```

Comment:
Maybe this makes more sense as Reviews.findByWine(wineId) or Wine.findReviewsById(wineId) - because right now we're making 2 DB calls - one for the wine, then another for its reviews